### PR TITLE
removing text inconsistency

### LINKED ISSUE #390: Text "use network to download content list." should be in the center of screen
screenshot_20180214-222149

### PR DESCRIPTION
**Pull request by CheshtaK**
Fixes #390

Changes: Fixed text inconsistency across the Device, Online and Downloading screens by changing the corresponding xml files.

Screenshots/GIF for the change:
Screenshot_20190827-201625 Screenshot_20190827-201633 Screenshot_20190827-201641

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,248 +1,30 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <option name="USE_SAME_INDENTS" value="true" />
```
Comment (macgills) [inline .idea/codeStyles/Project.xml]: Please do not delete these files, they are here to ensure that all developers share the same codestyles

Comment (CheshtaK) [reply .idea/codeStyles/Project.xml]: Sorry! I didn't realize. Could you please guide me on how to proceed? Do I need to clone the repository again or create a new branch?

Comment (macgills) [reply .idea/codeStyles/Project.xml]: you can revert this commit and then `cherrypick` your changes from the first commit.
Or if you don't want to do that just delete this branch, close this PR and open a new branch/pr after redoing the changes.

Comment (CheshtaK) [reply .idea/codeStyles/Project.xml]: Okay thanks!

**Pull request closed without merging**